src/compute.geometry/ResthopperEndpoints.cs

```diff
@@ -57,6 +59,23 @@ public static GH_Archive ArchiveFromUrl(string url)
             return null;
         }
 
+        static void SetDefaultTolerances(double absoluteTolerance, double angleToleranceDegrees)
+        {
+            Assembly assembly = Assembly.GetAssembly(typeof(Grasshopper.Utility));
```

Comment:
```
var utilityType = typeof(Grasshopper.Utility);
var method = utilityType.GetMethod(...)
```
less work and less error prone than digging around an assembly when you already know the type

Reply:
since absoluteTolerand and angleToleranceDegrees are by default 0, we should check and bail out of this function when either of them are 0